dist_tables/append_distribution.rst

```diff
@@ -1,167 +0,0 @@
-.. _append_distribution:
```

Comment:
Once that one is merged then a rebase will clean up the diff in this PR. Probably slipped in because I had some staged stuff from the other change.